validate/automated/nodectlCommands.md

```diff
@@ -0,0 +1,1858 @@
+import MacWindow from '@site/src/components/global/MacWindow';
+
+<head>
+  <title>nodectl command reference guide</title>
+  <meta
+    name="description"
+    content="nodectl command reference guide"
+    title="Command Reference Guide"
+  />
+  <style>{`
+    :root {
+      --doc-item-container-width: 60rem;
+    }
+  `}
+  </style>
+</head>
+
+
+
+
+
+<span style={{color: "purple", fontSize:"4em", fontWeight:"800"}}>nodectl</span>
+
+## Introduction
+
+This document compliments the **nodectl** [help command](#help) reference offered through execution of a Node running the **nodectl** utility. 
+
+
+
+
+
+### What is a switch and parameter?
+
+A command-line **switch** is a modifier that is added to the end of a command being executed by **nodectl**.
+
+It follows the `sudo nodectl <command>`.  
+
+When the command is requested and the `switch` added, it may be followed by specific items or "instructions" that are called **parameters**.
+
+#### Examples
+
+`sudo nodectl <command> <switch> <parameter>`
+
+`sudo nodectl <command> <switch> <parameter> <switch> <parameter>`
+
+`sudo nodectl <command> <switch> <parameter> <switch> <switch>`
+
+Some **switches** do not require a **parameter** afterwards.
+
+| As a simple example, the command
+```
+sudo nodectl status -p dag-l0
+```
+
+- The `status` is the **command**
+
+- The `-p` is a **switch**
+
+- The `dag-l0` is a **parameter**.
+
+:::note
+This reference guide will explorer the [status command](#status) in further detail; however, in the above example, the switch `-p` stands for "what profile would you like to explorer the status of?" and the parameter `dag-l0` is the profile in question we would like to review.
+:::
+
+:::note Final Note
+If a `switch` requires a `parameter`, it must be directly after the `switch` is supplied on the command line.  However, the order of the `switches` that do not require or require parameters does **not** matter.
+
+switch1 requires a parameter1, switch2 does not require a parameter.
+```
+sudo nodectl -switch1 parameter1 -switch2
+```
+is the same as
+```
+sudo nodectl -switch2 -switch1 parameter1
+```
+:::
+
+
+
+
+
+
+### What is pagination?
+
+The most common method of accessing your Node is through a [remote shell](../validator/sshkeyExplained.md).  When we issue a command within our remote shell, the output of a command may extend past the height of the terminal window.  In this case, **nodectl** will "page" the output by pausing when the output printed to the screen reaches the height of the window, before needing to scroll.
+
+It will offer you the option to quit or continue.
+
+Commands that create the need for pagination will generally offer a `-np` (*no pagination*) [switch](#what-is-a-switch-and-parameter) to disable pagination.
+
+<MacWindow>
+  Press any key or `q` to quit
+</MacWindow>
+
+
+
+
+## Command References
+---
+
+
+
+
+### <span style={{color:'green',fontSize:'1.3em'}}>help</span>
```

Comment:
No introducing custom colors. Work with Chad to add something to the stylesheet if it's really necessary.